business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class BusinessRequirementResource(models.Model):
+    _name = "business.requirement.resource"
+    _description = "Business Requirement Resource"
+
+    sequence = fields.Integer('Sequence')
+    description = fields.Char('Description', required=True)
+    product_id = fields.Many2one(
+        comodel_name='product.product',
+        string='Product',
+        required=False
+    )
+    uom_id = fields.Many2one(
```

Comment:
There should an onchange function to adapt the cost price according to the UOM (check the PO module)